docs/modules/usage_guide/pages/catalogs.adoc

```diff
@@ -0,0 +1,74 @@
+= Catalogs
+
```

Comment:
```suggestion
Catalogues are defined in their own resources and referenced from cluster objects.
```

Here we need a bit of an introductory sentence. It would also be good to link to the concept page from here